index.html

```diff
@@ -1,17 +1,145 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Restaurante</title>
-	<link rel="stylesheet" href="css/bootstrap.min.css">
-	<link rel="stylesheet" href="css/font-awesome.min.css">
-	<link rel="stylesheet" href="css/styles.css">
-</head>
-<body>
-
-	<!-- javascript	 -->
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-	<script src="js/bootstrap.min.js"></script>
-</body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
+		/>
+		<title>Restaurante</title>
+		<link rel="stylesheet" href="css/bootstrap.min.css" />
+		<link rel="stylesheet" href="css/all.min.css" />
+		<link rel="stylesheet" href="css/styles.css" />
+	</head>
+	<body>
+		<!-- NAV -->
+
+		<!-- nav.navbar.navbar-default>.container-fluid>(.navbar-header>(button[type="button"].navbar-toggle.collapsed[data-toggle=collapse][data-target="#mainNav"]>(span.icon-bar)*4)+a.navbar-brand>img[src=img/name-01.png])*1 -->
```

Comment:
you left commented out code here. was this intentional?

Reply:
This was intentional to review the emmet lines at a later date